### DISCUSSION TITLE
Phishing site hosted on verecel

### DISCUSSION THREAD

**Discussion by CharlesKerr** [vercel/community#6231] [Account Issue] [answered]
### Question

Please be advised, I have come across a crypto phishing site hosted on vercel. I'm just trying to let the Admins know, anyway one can trace the guy who publshed  it?

**Comment by amyegan** [comment, marked as answer]
Hey @CharlesKerr. Thanks for reporting this! If you haven't already, please send a copy of this report to [abuse@vercel.com](mailto:abuse@vercel.com) so it can be properly tracked in our system. I'll let the team know to be on the lookout for your email 🙇

**Reply by amyegan** [reply]
The team let me know they received your report and have responded. Thanks again for reporting it!
